app/helpers/submissions_helper.rb

```diff
@@ -9,4 +9,10 @@ def eligible_for_evaluation?(submission)
   def selected_to_advance?(submission)
     submission.judging_status.in?(%w[winner])
   end
+
+  def available_evaluators(submission)
+    submission.phase.evaluators -
+      submission.evaluators +
+      submission.evaluators.where("evaluator_submission_assignments.status" => ["unassigned"])
+  end
```

Comment:
Not a blocker, just a suggestion basically 
```suggestion
  # Phase evaluators not currently assigned/recused on the submission
  def available_evaluators(submission)
    unavailable_evaluators = submission.evaluators.where.not("evaluator_submission_assignments.status" => "unassigned")
    submission.phase.evaluators.where.not(id: unavailable_evaluators)
  end
```
This query could be even moved to the Submisssion model as `submission.available_evaluators`

Reply:
skip this for now, I can provide a commit to update it for you